README-InstrumentQ.md

```diff
@@ -11,21 +11,45 @@ When using fields in filter conditions and sorting, the raw field value will be
 When displaying the field, the field value's label will be used by default. This can be overridden
 using `:value` and `:label`, for example: `[instrument_1][choice_field]:value`
 
+When specifying *fields to display*, `:edit` can be appended to the field name to allow a value to

```

Comment:
Does this protect against SQL injection?  If so, perhaps also say in the readme file to re-assure people who use it.

Reply:
Field values are not being updated by direct database queries, but using built-in REDCap functions. Therefore SQL injection should not be a concern here as REDCap is handling the database updates (and performing logging etc.)